### ISSUE TITLE
FTBFS on arm64, armhf

### ISSUE THREAD

**Issue by mnalis** [mnalis/ironseed_fpc#87]
Fails to build from source: https://buildd.debian.org/status/package.php?p=ironseed

```
(9009) Assembling _paths_
(9009) Assembling utils_
(9009) Assembling ironseed
(9015) Linking is
is.pas(88) Warning: (9034) "crtbeginS.o" not found, this will probably cause a linking failure
is.pas(88) Warning: (9034) "crtendS.o" not found, this will probably cause a linking failure
is.pas(88) Fatal: (10026) There were 2 errors compiling module, stopping
Fatal: (1018) Compilation aborted
Error: /usr/bin/ppca64 returned an error exitcode
make[2]: *** [Makefile:81: is] Error 1
make[2]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_build: error: make -j1 "INSTALL=install --strip-program=true" prefix=/usr libdir=/usr/libexec/ironseed returned exit code 2
make[1]: *** [debian/rules:14: override_dh_auto_build] Error 25
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
make: *** [debian/rules:8: binary-arch] Error 2
```

see [fpc FAQ](https://wiki.lazarus.freepascal.org/Lazarus_Faq#I_receive_a_warning_during_the_linking_that_states:_Warning:_.22crtbeginS.o.22_.28or_.22crtendS.o.22.29_not_found) for possible reasons

**Comment by mnalis** [comment]
Failing compilation on warning/notes should only be done when developing. (as fixed in bda1037bb868a128c12c4242e890bb2aa0ad5de9 ?) 

Although, we shouldn't be getting those warnings, as the system path should be included.
Reported in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=978040

**Comment by mnalis** [comment]
we now also autocalculate the path in Makefile. Fixed.
